src/main/java/org/folio/circulation/infrastructure/storage/ServicePointRepository.java

```diff
@@ -200,6 +206,33 @@ public CompletableFuture<Result<Collection<ServicePoint>>> findServicePointsById
       .thenApply(r -> r.map(MultipleRecords::getRecords));
   }
 
+  public CompletableFuture<Result<Set<String>>> fetchPickupLocationServicePointsIds() {
```

Comment:
This method is so specific that I think it is unlikely that we will ever use it again. Can we make it more reusable by letting it just fetch all pickup locations and let the client service extract the IDs?

Reply:
done